Sources/IO/XML/XMLPolyDataReader/index.js

```diff
@@ -1,6 +1,7 @@
 import vtkXMLReader from 'vtk.js/Sources/IO/XML/XMLReader';
 import macro from 'vtk.js/Sources/macro';
 import vtkPolyData from 'vtk.js/Sources/Common/DataModel/PolyData';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
```

Comment:
sort alphabetically

Reply:
I'll take care of this along with the kw-web-suite update.